airflow/www/templates/airflow/dags.html

```diff
@@ -319,7 +320,7 @@ <h2>{{ page_title }}</h2>
                       {% if ds_info.total == 1 -%}
                         On {{ ds_info.uri[0:40] + '…' if ds_info.uri and ds_info.uri|length > 40 else ds_info.uri|default('', true) }}
                       {%- else -%}
-                      {{ ds_info.ready }} of {{ ds_info.total }} datasets updated
+                        <span class="js-datasets-ready">{{ ds_info.ready }}</span> of  <span class="js-datasets-total">{{ ds_info.total }}</span> datasets updated
```

Comment:
Are these new classes (`js-datasets-ready` and `js-datasets-total`) actually used?